docs/index.asciidoc

```diff
@@ -217,14 +214,14 @@ Close Idle clients after X seconds of inactivity.
 [id="plugins-{type}s-{plugin}-ecs_compatibility"]
 ===== `ecs_compatibility`
 
-* Value type is <<string,string>>
-* Supported values are:
-** `disabled`: unstructured connection metadata added at root level
-** `v1`: structured connection metadata added under ECS v1 compliant namespaces
-** `v8`: structured connection metadata added under ECS v8 compliant namespaces
-* Default value depends on which version of Logstash is running:
-** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
-** Otherwise, the default value is `disabled`.
+  * Value type is <<string,string>>
+  * Supported values are:
+    ** `disabled`: unstructured connection metadata added at root level
+    ** `v1`: structured connection metadata added under ECS v1 compliant namespaces
+    ** `v8`: structured connection metadata added under ECS v8 compliant namespaces
+  * Default value depends on which version of Logstash is running:
+    ** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+    ** Otherwise, the default value is `disabled`.
```

Comment:
The asciidoc formatting is correct as it was.  You don't even have to add the leading space to the first level bullets. Asciidoc automatically indents them for you. 
The second level bullets are tagged `**` and should be aligned with the left margin.

Reply:
yy it just read weird - the rest of the documented options had the whitespaces to aid while reading the source

Reply:
If you like the leading spaces before the first level bullets, I don't have a problem with that. They won't hurt anything. 

Asciidoc standard is to left align all bullets. I like following the standards because otherwise, people see things done differently and a new "pseudo standard" is born.